stripe/src/main/java/com/stripe/android/view/CardNumberEditText.kt

```diff
@@ -395,13 +401,24 @@ class CardNumberEditText internal constructor(
                 if (isComplete(wasCardNumberValid)) {
                     completionCallback()
                 }
+            }
+            // Partial card number entered and brand is not yet determine, but possible.
+            else if (isPossibleCardBrand(unvalidatedCardNumber.normalized)) {
+                isCardNumberValid = isValid
+                shouldShowError = true
             } else {
                 isCardNumberValid = isValid
-                // Don't show errors if we aren't full-length.
+                // Don't show errors if we aren't full-length and the brand is known.
+                // TODO (michelleb-stripe) Should set error message to incomplete, then in focus change if it isn't complete it will update it.
                 shouldShowError = false
             }
         }
 
+        private fun isPossibleCardBrand(cardNumber: String): Boolean {
```

Comment:
I don't think I'm clear on what "possible" means here

Reply:
0, 1 are not possible card brands, and a 3 could be either AMex or diners, etc.   So 3 is possible, but not 0 or 1

Reply:
The name here is actually wrong, should be isImpossibleCardBrand